### PR TITLE
adding cards to index.html

### PR DESCRIPTION
**Pull request by Arvindjitta**
I would add the basic cards in the ` index.html`  file, that has useful information listing all the community organisations projects, with the following information:

project name
project description
repo link
website link
tech stack
status

In large and extra large displays it looks like this 
lg-disc

-------------------------------------------------------------------------------------------------
In medium devices it looks like this
md-disc

--------------------------------------------------------------------------------------------------
In mobile devices it looks like this
sm-disc

Added Three icons 
 1. first icon is  for github profile/ personal website

 2. second icons redirects to repository/ github profile

 3.  third just added star icon (we can redirect to any important link in future pushes)

### CODE REVIEW

**Comment by eddiejaoude** [comment]
Please include a screenshot of how it looks in the page 😉

**Comment by eddiejaoude** [comment]
Its a good start, but it does not look ready to merge in yet?

**Comment by Arvindjitta** [comment]
> Its a good start, but it does not look ready to merge in yet?

yeah, i'll work on it

**Comment by eddiejaoude** [comment]
no problem, I will convert it to a `draft` PR for now

File: index.html [hunk 1]
```diff
@@ -229,5 +229,40 @@ <h2 class="text-4xl mb-8 font-semibold">
...
+                <h1 class="font-medium text-xl pt-8">awesome-github-profiles</h1>
+                <p class="font-light text-sm py-1">List of GitHub profiles that have awesome
+                    customisation, that you can use for inspiration.</p>
+                <ul class="lg:pb-5 lg:pt-2 ">
+                    <li class="py-1 flex flex-inline">
+                        <i class="material-icons">link</i>
+                        <a class="pl-2" href="">view repo</a>
+                    </li>
+                    <li class="py-1 flex flex-inline">
+                        <i class="material-icons">link</i>
```
Comment (eddiejaoude) [inline index.html:250]: These icons needs to be replaced with FontAwesome icons

Comment (Arvindjitta) [reply index.html:250]: yeah, i'll change it..

Comment (eddiejaoude) [reply index.html:250]: 👍

Comment (Arvindjitta) [reply index.html]: i have changed it to fontaswesome icons

Comment (eddiejaoude) [reply index.html]: ok great 👍

**Comment by eddiejaoude** [comment]
@BOLT04 the latest work has been pushed to this PR

File: index.html [hunk 2]
```diff
@@ -229,5 +229,118 @@ <h2 class="text-4xl mb-8 font-semibold">
...
+            <h1
+              class="font-medium text-lg sm:text-xl md:text-xl lg:text-xl xl:text-xl font-medium pt-2 sm:pt-1 md:pt-1 lg:pt-1">
+              Eddie Jaoude</h1>
+            <p
+              class="font-light text-xs sm:text-sm md:text-sm lg:text-sm xl:text-sm py-1 font-normal hidden   md:block lg:block break-words">
+              Fullstack open source DevRel! I love code and the community :). GitHub Star ⭐ program 🤓</p>
+            <div class=" hidden lg:inline-flex  md:inline-flex  flex flex-inline">
+              <div class="flex flex-inline">
+                <i class="fa fa-github-alt pr-1 pt-6 "
+                  style="font-size:18px; color:rgb(109, 71, 199); font-style:normal;"></i> </i>
```
Comment (eddiejaoude) [inline index.html:256]: These should be tailwind styles

Comment (Arvindjitta) [reply index.html:256]: @eddiejaoude okay, brother now where do I make these changes ??

Comment (eddiejaoude) [reply index.html:256]: New PR please 🤓

**Pull request merged**